core/src/HIP/Kokkos_HIP_BlockSize_Deduction.hpp

```diff
@@ -55,6 +55,26 @@
 namespace Kokkos {
 namespace Experimental {
 namespace Impl {
+
+template <typename DriverType, bool, int MaxThreadsPerBlock, int MinBlocksPerSM>
```

Comment:
So what's the point of the `bool`? it is never used.

Reply:
when we get around to implementing the constant launcher, it will be.  We could remove it for the moment if you;d prefer